tests/reg_tests/test_DVConstraints.py

```diff
@@ -668,11 +668,14 @@ def test_surfaceArea_box(self, train=False, refDeriv=False):
             )
 
     def test_projectedArea(self, train=False, refDeriv=False):
+        np.random.seed(37)
```

Comment:
Very minor, but NumPy suggests that it is better practice to use their generator rather than these functions to support legacy code. The suggested method creates a generator with `gen = np.random.default_rng(37)` and then generating random numbers with `gen.random()`.

Reply:
Yeah you're right, I was just being lazy and copying what's done elsewhere in these tests, will change it now

Reply:
The `default_rng.random` doesn't produce the same output as the old `np.random.rand` even with the same seed, so I had to re-train all the tests

Reply:
Well that’s frustrating….thanks for updating